pages/docs/community/glossary.md

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
Heyo, your directory structure is wrong because it's missing the `Community` content bucket folder. 😄 We are storing our Style Guide in the `Community` content bucket so your directory needs to be:
`asyncapi.com/docs/community/style-guide/glossary`.

Reply:
P.S. I didn't mention this before because we JUST merged our new `Community` content bucket into the docs this week... but now that we have a LIVE `Community` content bucket in the docs, we actually should be authoring community docs directly in the `/Community` repo. (We actually have similar behavior/process in place for tools and specification docs too! Tools and specification docs are actually stored in their repoes and then copies to our website repo docs. 

**I would work on your second draft and create a NEW PR in the community repo when you have it ready to review.** 

Please let me know if this doesn't make sense!

Reply:
I will need more explanation on this.
I currently have the following and the docs resides inside the 'Pages' folder as shown in the screenshots below.
<img width="377" alt="image" src="https://github.com/asyncapi/website/assets/33583060/ea5641ca-f341-403c-a9d4-f2ce8d894510">

second screenshot. 
<img width="381" alt="image" src="https://github.com/asyncapi/website/assets/33583060/9e0981b0-2483-42ec-a275-6e4dcd80c879">

Reply:
Yes, exactly! The sub-folder `community` inside the `/docs` directory is where the Style Guide will live.

That said, please note that your **glossary** document should be inside a NEW sub-folder of `Style Guide`. The URL for your document should be as follows: `asyncapi.com/docs/community/style-guide/glossary.`

Please let me know if this makes sense, we can always have another quick call too 💜

Reply:
<img width="240" alt="image" src="https://github.com/asyncapi/website/assets/33583060/7d43cace-78c7-4567-8da1-cb1f5638beff">

I noticed that, too and fixed it up. Thanks for calling that out.

Reply:
SOOOO CLOSE, @CynthiaPeter. Please note, however, that the sub-directory `style-guide` is missing a dash to separate the words. 

As noted above, the URL for your document should be as follows: asyncapi.com/docs/community/style-guide/glossary.

😄✌🏽

Reply:
Thank you, @alequetzalli 
I fixed these. 

However, is there a guide on how we decide what goes into the _section.md and index.md page?

And if you have the time, I'd like to connect and have a call to talk about this and the onboarding guide. I have some drafts and ideas, but I am not exactly sure how to go about it.

Reply:
Yes, there is def best practices on what you need to place in both those files. Let's go over this again in a call 😄.

I see you sent me a DM to pick a day/time for that call, so I will follow up with you offline to help unblock you.